cellprofiler/gui/figure.py

```diff
@@ -1493,6 +1493,15 @@ def subplot_imshow_color(self, x, y, image, title=None, normalize=False, rgb_mas
         if rgb_mask is None:
             rgb_mask = [1, 1, 1]
 
+        # Truncate multichannel data that is not RGB(A) (5+ channel data) and display it as RGB.
+        if image.shape[2] > 4:
+            logger.warn(
+                "Multichannel display is only supported for RGB(A) (3- or 4-channel) data."
+                " Input image has {:d} channels. Only the first 3 channels will be displayed.".format(image.shape[-1])
```

Comment:
Perhaps this could be `Only the first 3 channels will be displayed as RGB`?

Reply:
Good suggestion! I'll edit the message.